### DISCUSSION TITLE
Add Wifi Hotspot

### DISCUSSION THREAD

**Discussion by actraiser30** [Exarkuniv/RetroPie-Extra#219] [Ideas]
Hello

Is possible to add this to your script?

It creates automatic a wifi hotspot 

https://github.com/arm358/Raspberry-Pi-Automated-WiFi-Access-Point

Instructions 

git clone https://github.com/arm358/Raspberry-Pi-Automated-WiFi-Access-Point

cd Raspberry-Pi-Automated-WiFi-Access-Point

sudo chmod +x setup.sh

./setup.sh

Thanks

**Comment by Exarkuniv** [comment]
If you want to try your hard to making a script go for it. 
I don't see it getting much use of any. Other then you since you had the suggestion. 

But I will add it to the back burner. I have a other things I believe that will get more use
